src/Resources/config/app/routing/admin/custom_block.yml

```diff
@@ -0,0 +1,51 @@
+lakion_sylius_cms_admin_custom_block:
+    resource: |
+        alias: lakion_sylius_cms.custom_block
+        section: admin
+        templates: SyliusAdminBundle:Crud
+        except: ['show', 'update', 'delete']
+        redirect: index
+        grid: lakion_sylius_cms_admin_custom_block
+        vars:
+            all:
+                templates:
+                    form: "@LakionSyliusCms/Admin/CustomBlock/_form.html.twig"
+            index:
+                icon: font
+    type: sylius.resource
+
+lakion_sylius_cms_admin_custom_block_show:
+    path: /custom-blocks{id}/show
```

Comment:
That comes form Symfony CMF's BlockBundle, but I'll try using unique `name` instead.